Week1/homework/js-exercises/compareArray.js

```diff
@@ -0,0 +1,17 @@
+var names=['Nur','gul','tur','yuri'];
+var numbers=[1,2,3,4,5,6,7];
+
+console.log('The length of the array is ' + names.length);
+// console.log(names.length);
+console.log('The length of the array is ' + numbers.length);
+// console.log(numbers.length);
+
+let compare = 'names.length === numbers.length';
+compare=false;
```

Comment:
I think you made a mistake here, can you take a look at [this](https://developer.mozilla.org/nl/docs/Web/JavaScript/Equality_comparisons_and_sameness) and see if you can fix it.